src/Illuminate/Encryption/CipherMethods.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Illuminate\Encryption;
+
+
+trait CipherMethods
+{
+    abstract public function getLength() :int;
```

Comment:
Please don't use return type annotations within the code like this. The framework intentionally avoids them.